package.json

```diff
@@ -23,8 +23,11 @@
     "pretest": "npm run lint",
     "test": "npm run jest",
     "jest": "cross-env MOCK_REDIS=1 jest",
-    "jest-watch":"cross-env MOCK_REDIS=1 jest --watch",
-    "start": "node src",
+    "jest-watch": "cross-env MOCK_REDIS=1 jest --watch",
+    "start": "node src/backend",
+    "server": "node src/backend/web/server",
```

Comment:
Do we need server because of webhint? What exactly would adding webhint do in this PR?

Reply:
based on the running I just did  without it I would argue it  helps set the setup, as it seems to go a bit off the rails without, also since we are  reverting  items to previous version I rather be safe